media/css/firefox/firstrun/firstrun_quantum.less

```diff
@@ -135,6 +135,11 @@ body {
     border: none;
     position: relative;
     width: 318px;
+    &.treatment {
```

Comment:
I didn't want to affect styling of the original page, so I limited this change to the treatment group.

Reply:
Makes sense! I suggest here you add a comment with a reference to the bug number, so our future selves can work out why this is here :)

Reply:
Done